### ISSUE TITLE
Cannot place cart in world (Version 2.1.0.62)

### ISSUE THREAD

**Issue by dinoracer01** [modmuss50/SC2#40]
Not entirely sure if it's just some of the modules aren't completely ported or if it's a bug, but it says I can't place it in the world due to an issue with the modules on it. Here's the "relevant"(?) part of the log(also don't know how to do spoilers):

#347]  [17:53:47] [Server thread/ERROR] [stevescarts]:  --------------- Broken cart info --------------- 
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: {Data2:100b,Modules:[21 bytes],CartVersion:2b}
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.hull.ModuleCheatHull
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.engines.ModuleCheatEngine
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.workers.tools.ModuleDrillGalgadorian
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.realtimers.ModuleSeat
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.realtimers.ModuleAdvControl
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.workers.ModuleRailerLarge
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.workers.ModuleBridge
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.realtimers.ModuleArcade
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.workers.ModuleHydrater
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.storages.tanks.ModuleCheatTank
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.storages.chests.ModuleInternalStorage
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.storages.chests.ModuleInternalStorage
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.addons.ModuleCreativeSupplies
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.addons.ModuleLabel
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.addons.ModuleShield
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.addons.ModuleDrillIntelligence
[B#347]  [17:53:47] [Client thread/INFO]: [CHAT] The cart failed to be placed into the world, this is due to an issue with one or more modules. Please post your log on the issue tracker here: §9 https://github.com/modmuss50/SC2/issues
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.addons.ModuleHeightControl
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.addons.ModulePowerObserver
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.addons.ModuleMelterExtreme
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.addons.ModuleCreativeIncinerator
[B#347]  [17:53:47] [Server thread/ERROR] [stevescarts]: --- vswe.stevescarts.modules.addons.ModuleColorizer

**Comment by dinoracer01** [comment]
here's the world startup log (the entire game log is too big for pasting anywhere)
http://pastebin.com/fd5Sb0cR

**Comment by JellyBlade** [comment]
I'm experiencing the same issue. The weird thing is that I had no issue until very recently, when I loaded up my world (after updating a few of my mods... woops), and noticed my treefarm cart disappeared. I built myself a new one, and it threw the broken cart errors at me. At line 38536 (if you have notepad++, or another text editor that supports lines), there's a duplicate entity id error thrown for steve's carts.
[fml-client-latest.txt](https://github.com/modmuss50/SC2/files/715544/fml-client-latest.txt)
